source/IrcDotNet/IrcEventArgs.cs

```diff
@@ -10,11 +10,38 @@
 
 namespace IrcDotNet
 {
-    /// <summary>
-    ///     Provides data for the <see cref="IrcClient.ChannelListReceived" /> event.
-    /// </summary>
-    /// <threadsafety static="true" instance="false" />
-    public class IrcChannelListReceivedEventArgs : EventArgs
+	/// <summary>
```

Comment:
Yeah, definitely a spaces vs. tabs issue here. I don't care which you want to go with, but let's have it consistent throughout the project.